lib/exnumerator/atom_list_enum.ex

```diff
@@ -0,0 +1,40 @@
+defmodule Exnumerator.AtomListEnum do
+  def cast(values, term) when is_binary(term) do
+    atom_term = String.to_atom(term)
+    cast(values, atom_term)
```

Comment:
What is your opinion about this approach:

```
term
|> String.to_atom()
|> cast(values)
```

and switch arguments order in functions?

Reply:
The arguments are following everyone else.

I think this is something that Elixir as a pure functional language will need to deal with or create some library that does those common situations of flipping the order.